tests/storage/test_delete.py

```diff
@@ -0,0 +1,65 @@
+import hashlib
+import os
+import glob
+import traceback
+import re
+import binascii
+import logging
+import sys
+
+sys.path.append("..")
+import time
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from TestBase import Test
+import argparse
+
+extension = ".txt"
```

Comment:
What is this used for?
Same question for the other 'extension' variables in the other tests